py-json/interop_connectivity.py

```diff
@@ -533,33 +1025,76 @@ async def set_port(self, port_list=[]):
             port = port_data['sta_name']
             interest = port_data['interest']
-
+            # report_timer = port_data['report_timer']
+            current_flags = port_data['current_flags']
             os = port_data['os']
-            if (os in ['Apple', 'Lin']):
-                current_flags = port_data['current_flags']
+            if (os == 'Lin'):
                 data = {
                     'shelf': shelf,
                     'resource': resource,
                     'port': port,
-                    'current_flags': current_flags,
-                    'interest': interest,
+                    "report_timer": 1,
+                    'current_flags': 2147483648,
+                    'interest': 92291074,
                     'mac': self.mac
                 }
-            elif (os == 'Win'):
-                report_timer = port_data['report_timer']
-                current_flags = port_data['current_flags']
+            else:
                 data = {
                     'shelf': shelf,
                     'resource': resource,
                     'port': port,
-                    'report_timer': report_timer,
-                    'current_flags': current_flags,
-                    'interest': interest
+                    "report_timer": 1,
+                    'current_flags': 2147483648,
+                    'interest': 92291074,
                 }
             data_list.append(data)
 
         url = 'http://{}:{}/cli-json/set_port'.format(self.lanforge_ip, self.port)
+        loop = asyncio.get_event_loop()
+        tasks = [loop.run_in_executor(None, self.post_data, url, data) for data in data_list]
 
+        # Use asyncio.gather to await the completion of all tasks
+        results = await asyncio.gather(*tasks)
+
+    async def set_radio(self, port_list=[]):
+        logger.info("SET PORT LAPTOP")
+        if (port_list == []):
+            logger.info('Port list is empty')
+            return
+
+        data_list = []
+        for port_data in port_list:
+            shelf = port_data['shelf']
+            resource = port_data['resource']
+            port = port_data['sta_name']
+            interest = port_data['interest']
+            # report_timer = port_data['report_timer']
+            current_flags = port_data['current_flags']
+            os = port_data['os']
+            if (os == 'Lin'):
+                data = {
+                    'shelf': shelf,
+                    'resource': resource,
+                    'port': 'wiphy0',
+                    "report_timer": 1,
+                    'current_flags': 0,
+                    'interest': 92291074,
+                    'mac': self.mac
+                }
+            else:
+                data = {
+                    'shelf': shelf,
+                    'resource': resource,
+                    'port': port,
+                    "report_timer": 1,
+                    'current_flags': 2147483648,
+                    'interest': 92291074,
```

Comment:
Please comment flags

Reply:
@shubhcandela Can you or another developer add comments for the current and interest flags set here?